### PR TITLE
Forgot another 2024

### PR DESCRIPTION
**Pull request by nabobalis**
Some minor text changes.

### CODE REVIEW

File: gsoc/index.md [hunk 1]
```diff
@@ -6,19 +6,17 @@ permalink: /gsoc/
...
-[Background on GSoC: start here!](./background.html)
+[Background on GSoC: Start Here!](./background.html)
 
-## GSoC/Open Astronomy 2023
+## GSoC & Open Astronomy 2024
 
-OpenAstronomy is an umbrella organisation which collects project ideas
-from any of its members. OpenAstronomy has been a GSOC mentoring organisation since 2016 and
-it's applying to participate in 2023.
+OpenAstronomy is an umbrella organisation which collects project ideas from any of its members.
```
Comment (nabobalis) [inline gsoc/index.md:13]: I couldn't help myself and did one sentence per line. I can undo it.

Comment (bsipocz) [reply gsoc/index.md:13]: I don't have a preference beyond not having the full paragraph in one giga line, but one sentence per line is totally fine by me. Thanks!

**Review by bsipocz — APPROVED** [review]
Looks all good, thanks. We'll need to populate the admins section, too, but I'll first ping people on matrix about it.

File: gsoc/index.md [hunk 2]
```diff
@@ -6,19 +6,17 @@ permalink: /gsoc/
...
+OpenAstronomy has been a GSOC mentoring organisation since 2016 and it's applying to participate in 2024.
 
-Contributors applications to OpenAstronomy projects follows the same
-rules as the [Python Software Foundation] and the [GSoC Contributor Guide],
-we have [also our own guide to a good application for OpenAstronomy][OpenAstronomy Contributor Guide].
+Contributors applications to OpenAstronomy projects follows the same rules as the [Python Software Foundation] and the [GSoC Contributor Guide].
+We have [also our own guide to a good application for OpenAstronomy][OpenAstronomy Contributor Guide].
 
-All the student blogs are collected in the [OpenAstronomy Universe] site.
+All student blogs are collected in the [OpenAstronomy Universe] site.
```
Comment (nabobalis) [inline gsoc/index.md:19]:
```suggestion
All student blogs are collected on the [OpenAstronomy Universe] site.
```

**Comment by nabobalis** [comment]
Ah I saw that was empty, I can add the same as last year if you would like?

**Comment by bsipocz** [comment]
Thanks @nabobalis!

**Comment by bsipocz** [comment]
> Ah I saw that was empty, I can add the same as last year if you would like?

Let's see what the others say first.

**Pull request merged**